### PR TITLE
DBZ-2693 using db2 image with ocp entry-point

### PR DESCRIPTION
**Pull request by jcechace**
Complementary PR to https://github.com/debezium/debezium-examples/pull/135

### CODE REVIEW

**Comment by jcechace** [comment]
@jpechane  can the 1.2 image be updated or should I switch to some other tag? e.g. 1.2-testing to sth.

**Comment by jpechane** [comment]
@jcechace The 1.2 image can be updated

**Review by jpechane — APPROVED** [review]
@jcechace Approved, can be merged right after the parent is is merged

**Pull request merged**
